openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-autonomic/src/tools/Autonomic/Tabs/Triggers/TriggerOperand.vue

```diff
@@ -28,59 +28,63 @@
       class="mt-1"
       :data-test="`trigger-operand-${order}-type`"
       :items="operandTypes"
-    >
-      <template v-slot:item="{ item, on, attrs }">
-        <v-list-item
-          v-on="on"
-          v-bind="attrs"
-          :data-test="`trigger-operand-${order}-type-${item}`"
-        >
-          <v-list-item-content>
-            <v-list-item-title>{{ item }}</v-list-item-title>
-          </v-list-item-content>
-        </v-list-item>
-      </template>
-    </v-select>
+    />
     <div v-if="operandType === 'ITEM'">
       <v-row class="ma-0">
         <v-radio-group
-          v-model="itemValue"
+          v-model="valueType"
           class="px-2"
           row
-          @change="itemValueSelected"
+          @change="valueTypeSelected"
         >
-          <v-radio
-            label="RAW"
-            value="RAW"
-            :data-test="`trigger-operand-${order}-raw`"
-          />
-          <v-radio
-            label="CONVERTED"
-            value="CONVERTED"
-            :data-test="`trigger-operand-${order}-converted`"
-          />
+          <v-radio label="RAW" value="RAW" />
+          <v-radio label="CONVERTED" value="CONVERTED" />
+          <v-radio label="FORMATTED" value="FORMATTED" />
+          <v-radio label="WITH_UNITS" value="WITH_UNITS" />
         </v-radio-group>
       </v-row>
-      <target-packet-item-chooser vertical choose-item @on-set="itemSelected" />
+      <target-packet-item-chooser
+        vertical
+        choose-item
+        @on-set="itemSelected"
+        :initialTargetName="targetName"
+        :initialPacketName="packetName"
+        :initialItemName="itemName"
+      />
     </div>
     <div v-if="operandType === 'FLOAT'">
       <v-text-field
         label="Input Float Value"
         type="number"
+        hint="Press 'Enter' / 'Return' to commit value"
```

Comment:
Couldn't really make @keypress work so I just implemented @change and added the hint to press enter. It also works if you simply click somewhere other than the input.